config/crd/bases/modelregistry.opendatahub.io_modelregistries.yaml

```diff
@@ -149,15 +149,34 @@ spec:
                     description: True if skipping database instance creation during
                       ML Metadata service initialization. By default, it is false.
                     type: boolean
-                  sslCert:
-                    description: This parameter specifies the file name of the client
-                      SSL certificate, replacing the default ~/.postgresql/postgresql.crt.
-                    type: string
-                  sslKey:
-                    description: This parameter specifies the location for the secret
-                      key used for the client certificate. It can either specify a
-                      file name that will be used instead of the default ~/.postgresql/postgresql.key
-                    type: string
+                  sslCertificateSecret:
+                    description: This parameter specifies the Kubernetes Secret name
+                      and key of the client SSL certificate.
+                    properties:
+                      key:
+                        description: Key name in secret
+                        type: string
+                      name:
+                        description: Kubernetes secret name
+                        type: string
+                    required:
+                    - key
+                    - name
+                    type: object
+                  sslKeySecret:
+                    description: This parameter specifies the Kubernetes Secret name
+                      and key used for the client certificate SSL secret key.
+                    properties:
+                      key:
+                        description: Key name in secret
+                        type: string
+                      name:
+                        description: Kubernetes secret name
+                        type: string
+                    required:
+                    - key
+                    - name
+                    type: object
                   sslMode:
```

Comment:
If `sslMode=required` is not working, would it make sense to **_not_** have it in our options (adding comment to upstream `https://github.com/google/ml-metadata/issues/187#issue-2005410819` issue?)

I understand that is orthogonal to this PR scope, just noticed thought about mentioning